reframe/frontend/loader.py

```diff
@@ -115,7 +115,7 @@ def prefix(self):
     def recurse(self):
         return self._recurse
 
-    def load_from_module(self, module):
+    def load_from_module(self, module, *args, **kwargs):
```

Comment:
As mentioned in the previous comment, if we pass the `set_check_attributes` as an argument to the constructor of the loader, there is no need for all these `*args, **kwargs` here. The `load_from_module` could just use that attribute whenever needed.